lib/service/Service.ts

```diff
@@ -162,7 +162,8 @@ class Service extends EventEmitter {
    * Cancel placed order from the orderbook.
    */
   public cancelOrder = async ({ orderId, pairId }: { orderId: string, pairId: string }) => {
-    return this.orderBook.removeOwnOrder(orderId, pairId);
+    this.pool.broadcastOrderInvalidation(orderId, pairId);
+    return this.orderBook.removeOwnOrder(orderId, pairId).toString();
```

Comment:
Why are you using `toString()` and not `JSON.stringify()`?

Reply:
Considering it's just a boolean, I thought it doesn't make much of a difference